src/stdune/list.mli

```diff
@@ -78,3 +78,5 @@ val for_all2 :
   -> 'b list
   -> f:('a -> 'b -> bool)
   -> (bool, [ `Length_mismatch ]) result
+
+val min_by : 'a list -> f:('a -> 'a -> Ordering.t) -> 'a option
```

Comment:
What about just:

```suggestion
val min : 'a list -> compare:('a -> 'a -> Ordering.t) -> 'a option
```

?

That'd be consistent with the other functions.

Reply:
I renamed it.